.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - core/v*.*.*
+      - client/v*.*.*
+
+concurrency:
+  group: release-${{ github.ref }}
+
+jobs:
+  create-github-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
seems useless?

Reply:
Add this so that we don't need to clone the whole repo.